<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel de usuario</string>
  </property>
  <widget class="QPushButton" name="configButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuración</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoffButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar sesión</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emailButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar correo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar usuario</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
